src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -72,7 +70,7 @@ public class StudyController {
     private final VoltageInitService voltageInitService;
     private final LoadFlowService loadflowService;
     private final CaseService caseService;
-    private final ActuatorHealthService actuatorHealthService;
+    private final RemoteServices remoteServices;
```

Comment:
After discutions with team lead, let's rename this to RemoteServicesMonitor instead.